tests/test_langscript_filter.py

```diff
@@ -0,0 +1,15 @@
+
+
+from mteb.abstasks.languages import LangScriptFilter
+
+def test_langscript_filter():
+
+    ls_filter = LangScriptFilter.from_languages_and_scripts(
+        languages=["fra"],
+        scripts=None
```

Comment:
Nice! Would be great to include scripts as well.

Reply:
I have expanded the test cases to both cover script and other special cases